packages/components/organisms/f-checkout/src/demo/checkout-delivery.json

```diff
@@ -12,15 +12,16 @@
     "from": "2020-01-01T00:00+00:00",
     "to": "2020-01-01T00:00+00:00"
   },
-  "address": {
-    "lines": [
-      "1 Bristol Road",
-      "Flat 1",
-      "",
-      "Somerset"
-    ],
-    "locality": "Bristol",
-    "postalCode": "BS1 1AA"
+  "location": {
```

Comment:
Should you be updating [this test](https://github.com/justeat/fozzie-components/blob/66b3b03b1ba63290953610215cf65e1d109f0793/packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js#L161-L170) (which refers to `address` instead of `location` as part of the updates? It worries me that this test is still passing - which might imply it isn't testing what it is intended to be testing.

Reply:
Everything we store in state works the same as before. I just extract the address from the location object before it reaches the Vuex store. So all component/module tests can remain the same.

Reply:
I will update this test to be more clear with the new API schema though

Reply:
Updated :)